src/index.html

```diff
@@ -0,0 +1,19 @@
+<!-- index.html -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <!-- Include header -->
+  <include src="./components/header/header.html"></include>
```

Comment:
We can rename `header.html` into `index.html` inside the `header` folder
`<include src="./components/header/index.html"></include>`

Reply:
i solved it